source/agora/test/Base.d

```diff
@@ -99,6 +100,25 @@ shared static this()
     Runtime.extendedModuleUnitTester = &customModuleUnitTester;
 }
 
+void testAssertHandler (string file, ulong line, string msg) nothrow
+{
+    printf("[%.*s:%llu]================Assertion thrown during test: %.*s\n",
+            cast(int) file.length, file.ptr, line, cast(int) msg.length, msg.ptr);
```

Comment:
This makes for fairly long lines. If your intention is to have a clear "header" / separation, perhaps a 60 `=` characters (or any other character that lends itself to the task) would be better, followed by a `printf("[%.*s:%llu] Assertion failure: %.*s\n", ...);`

Reply:
Updated